### PR TITLE
Use system wide memcached.so

### PR DESCRIPTION
**Pull request by tvlooy**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | WiP
| License       | MIT

requested by https://github.com/symfony/symfony/pull/29624#issuecomment-457382732

let's see what travis is going to do with this ...

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -188,30 +188,36 @@ before_install:
               tfold ext.mongodb tpecl mongodb-1.6.0alpha1 mongodb.so $INI
           fi
 
+          echo extension = memcached.so >> $INI
```
Comment (nicolas-grekas) [inline .travis.yml]: should be after the if I suppose, see failures

Comment (tvlooy) [reply .travis.yml]: wheps

File: .travis.yml [hunk 2]
```diff
@@ -189,29 +189,36 @@ before_install:
...
+                  ./configure --with-libmemcached-dir=/opt/libmemcached/${LIBMEMCACHED_VERSION}
+                  make
+                  cp modules/memcached.so $ext_cache
+                  cd -
+              fi
 
-          echo extension = memcached.so >> $INI
+              echo extension = $ext_cache >> $INI
+          else
+              echo extension = memcached.so >> $INI
```
Comment (nicolas-grekas) [inline .travis.yml]: can't we use always this line outside of the if and remove the one before?

Comment (tvlooy) [reply .travis.yml]: I don't think so. The first is from cache and the second is the system wide. The cache is not in the ini path ... Let's try to find out

File: .travis.yml [hunk 3]
```diff
@@ -189,28 +189,32 @@ before_install:
...
+                  cd memcached-${EXTMEMCACHED_VERSION}
+                  phpize
+                  ./configure --with-libmemcached-dir=/opt/libmemcached/${LIBMEMCACHED_VERSION}
+                  make
+                  cp modules/memcached.so $ext_cache
+                  cd -
+              fi
           fi
-
           echo extension = memcached.so >> $INI
```
Comment (nicolas-grekas) [inline .travis.yml]: ok, got it, should be inside the previous if (but not inside the nested if), isn't it?

**Comment by tvlooy** [comment]
@nicolas-grekas this is better

**Comment by nicolas-grekas** [comment]
Awesome! Could you move the line back to the `php.ini configuration` section?

**Comment by nicolas-grekas** [comment]
Thank you @tvlooy.

**Pull request merged**